src/js/containers/router/RouterRoutes.js

```diff
@@ -35,6 +34,7 @@ const AboutTheDataPage = React.lazy(() => import('components/aboutTheData/AboutT
 const AgencyDetailsPage = React.lazy(() => import('components/aboutTheData/AgencyDetailsPage').then((comp) => comp));
 const ErrorPage = React.lazy(() => import('components/errorPage/ErrorPage').then((comp) => comp));
 const SubmissionStatisticsDataSources = React.lazy(() => import('components/aboutTheData/DataSourcesAndMethodologiesPage').then((comp) => comp));
+const DataDictionaryPage = React.lazy(() => import('components/dataDictionary/dataDictionaryPage').then((comp) => comp));
```

Comment:
this file needs to be upper case

`DataDictionaryPage`

Reply:
done